server/schedule/operator_controller_test.go

```diff
@@ -663,7 +663,9 @@ func (t *testOperatorControllerSuite) TestAddWaitingOperator(c *C) {
 	cluster := mockcluster.NewCluster(config.NewTestOptions())
 	stream := hbstream.NewTestHeartbeatStreams(t.ctx, cluster.ID, cluster, false /* no need to run */)
 	controller := NewOperatorController(t.ctx, cluster, stream)
-
+	cluster.AddLabelsStore(1, 1, map[string]string{"host": "host1"})
```

Comment:
Same storeID?

Reply:
Here is function sign:
```golagn
AddLabelsStore(storeID uint64, regionCount int, labels map[string]string) 
```

Reply:
Yeah, so you invoked with `storeID=1` for three times?

Reply:
nice catch